packages/mdc-linear-progress/README.md

```diff
@@ -52,6 +53,19 @@ npm install @material/linear-progress
 </div>
 ```
 
+### Accessibility
+
+Progress bars conform to the [WAI-ARIA Progressbar Specification](https://www.w3.org/TR/wai-aria/#progressbar). The supported ARIA attributes for this progress bar are:
+
+| Attribute | Description |
+| --------- | ----------- |
+| `aria-label` | Label indicating how the progress bar should be announced to the user. |
+| `aria-valuemin` | The minimum numeric value of the progress bar, which should always be `0`. |
+| `aria-valuemax` | The maximum numreic value of the progress bar, which should always be `1`. |
```

Comment:
typo: numeric

Reply:
Fixed and squashed everything into one commit.